### PR TITLE
Fixed PFAnalytics for watchOS.

### PR DESCRIPTION
**Pull request by nlutsenko**
Contributes to #179

### CODE REVIEW

File: Parse/PFAnalytics.m [hunk 1]
```diff
@@ -25,9 +25,11 @@ @implementation PFAnalytics
 ///--------------------------------------
 
 + (BFTask *)trackAppOpenedWithLaunchOptions:(NSDictionary *)launchOptions {
-#if TARGET_OS_IPHONE
+#if TARGET_OS_WATCH
+    NSDictionary *userInfo = nil;
+#elif TARGET_OS_IOS
     NSDictionary *userInfo = launchOptions[UIApplicationLaunchOptionsRemoteNotificationKey];
-#else
+#elif TARGET_OS_MAC
```
Comment (richardjrossiii) [inline Parse/PFAnalytics.m]: Seeing as how this is set to true for all iOS derivative platforms (e.g. tvOS as well). I wonder if we can come up with a better macro here...

Comment (nlutsenko) [inline Parse/PFAnalytics.m]: I can define PF_TARGET_OS_OSX and combine !TARGET_OS_WATCH, !TARGET_OS_IOS and eventually !TARGET_OS_TV.
What do you think?

Comment (richardjrossiii) [inline Parse/PFAnalytics.m]: We can do it that way, sure. I mean, we're still screwed when apple releases a new platform, but I think it's a bit less misleading that way.

Comment (nlutsenko) [inline Parse/PFAnalytics.m]: Great, did you have something specific in mind?

Comment (richardjrossiii) [inline Parse/PFAnalytics.m]: I wish I did, but I can't think of anything off of the top of my head. Just stick with `PF_TARGET_OS_OSX`, and we can update that macro if we have any brilliant ideas on how to make it better later.

**Pull request merged**